backend/src/main/java/com/soen341/instagram/service/impl/PictureService.java

```diff
@@ -133,4 +140,28 @@ private BufferedImage compressImage(BufferedImage image) {
         compressedImage.createGraphics().drawImage(image, 0, 0, Color.WHITE, null);
         return compressedImage;
     }
+    
+    // like service
+	public int likePicture(final String pictureId) {
```

Comment:
The indentations of `likePicture` and `unlikePicture` are wrong

Reply:
same as above, i do not see anything wrong in the indentation on eclipse, maybe it's a thing on the online viewer
![image](https://user-images.githubusercontent.com/45129406/76909292-3c4d7300-6881-11ea-9134-dfbcfc93d55e.png)

Reply:
ok so the problem lies with the fact that its using tabs and they're defined to be 4 spaces wide in eclipse while git shows tabs 6 wide